delivery_package_number/wizard/stock_inmediate_transfer.py

```diff
@@ -9,6 +9,14 @@ class StockImmediateTransfer(models.TransientModel):
     number_of_packages = fields.Integer(
         help="Set the number of packages for this picking(s)",
     )
+    ask_number_of_packages = fields.Boolean(compute="_compute_ask_number_of_packages")
+
+    @api.depends("pick_ids")
```

Comment:
Not sure if this is needed, as pick_ids won't change in the lifetime of the wizard...